test/multiply_test.cc

```diff
@@ -375,6 +383,55 @@ template <class Routine, class WriteC> void TestMultiply(Index A_rows, Index wid
    int_tolerance, float_tolerance, MSE_float_tolerance, MSE_int_tolerance);
 }
 
+//Code duplication may be avoided through some use of variadic templates, as the different WriteC symbols
+//Require different number of arguments. I don't think the refactoring is worth it.
+template <class Routine, class WriteC> void TestMultiplyBias(Index A_rows, Index width, Index B_cols,
```

Comment:
I think we really shouldn't pass WriteC as a template argument and then create and an object of that type with assumption it's BiasAddUnquantizeC. We have the same problem in TestMultiply so I quess it's just copy paste issue so please add some fixto comment or fix it if you want.

Reply:
You're right. I was hoping initially to be able to further template the code, but that's not possible. The WriteC template argument should be removed from all of the TestX functions. @kpu any objections?